dev/schema.js

```diff
@@ -77,6 +77,46 @@ module.exports = {
 			hint: "Minimum 6 characters",
 			styleClasses: "half-width",
 			validator: validators.string			
+		},
+
+		{
+			type: "vueMultiSelect",
+			label: "Skills (vue-multiSelect field)",
+			model: "skills",
+			required: true,
+			multiSelect: true,
+			selectOptions: {
+				// id:25,
+				// key:"name",
+				// label: "name",
+				searchable:true,
+				clearOnSelect:true,
+				hideSelected:true,
+				// maxHeight:300,
+				// allowEmpty:true,
+				// resetAfter:false,
+				closeOnSelect: true,
+				// customLabel:function(){return ""},
+				taggable:true,
+				tagPlaceholder:'Press enter to create a tag',
```

Comment:
I'm also adding `onSearch` (from `onSearchChange`) since it is a very useful function (to create dynamics option list for country and all kind of filtering).